Bottom: 867d58096ff5954f8f9733c9dd129915d7d5ad4c
Top:    e82e710c730c9e85c4d5f08526b8529ae34a340d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-10-01 12:31:59 -0500

Refresh of refresh-temp

---
diff --git a/ansible-playbook/group_vars/all/nexus3.yml b/ansible-playbook/group_vars/all/nexus3.yml
index f0e2f118e..7267f9f52 100644
--- a/ansible-playbook/group_vars/all/nexus3.yml
+++ b/ansible-playbook/group_vars/all/nexus3.yml
@@ -3,9 +3,8 @@ nexus_networks:
   - name: '{{ docker_network_frontend }}'
     aliases:
       - '{{ nexus_hostname }}'
-docker: true
-nexus_default_port: 80
-public_hostname: '{{ nexus_hostname }}'
+nexus_host_docker: true
+nexus_default_port: 8081
 
 nexus_email_server_enabled: true
 nexus_email_server_host: "mail.novatech-llc.com"
diff --git a/roles/nexus3/defaults/main.yml b/roles/nexus3/defaults/main.yml
index 992be594b..e37498dcb 100644
--- a/roles/nexus3/defaults/main.yml
+++ b/roles/nexus3/defaults/main.yml
@@ -31,6 +31,8 @@ nexus_backup_keep_rotations: 4  # Keep 4 backup rotation by default (current + l
 
 # Nexus purge procedure:
 # run ansible-playbook example.yml -e "purge=true"
+nexus_purge: false
+nexus_host_docker: false
 
 # Nexus default properties
 nexus_default_port: 8081
